Makefile

```diff
@@ -19,28 +19,23 @@ NITRITE_CMD=cmd/nitrite/main.go
 .PHONY: test-main
 test-main: tidy
 	@cat $(TESTDATA)/nitro_attestation.b64 | \
-		go run $(NITRITE_CMD) -attestation - 1>/dev/null
+		go run $(NITRITE_CMD) 1>/dev/null
 	@cat $(TESTDATA)/nitro_attestation_debug.b64 | \
-		go run $(NITRITE_CMD) -attestation - -allowdebug 1>/dev/null
+		go run $(NITRITE_CMD) -allowdebug 1>/dev/null
 	@$(shell cat $(TESTDATA)/nitro_attestation_debug.b64 | \
-    	go run $(NITRITE_CMD) -attestation - 2>/dev/null)
+    	go run $(NITRITE_CMD) 2>/dev/null)
 	@if [ $(.SHELLSTATUS) -eq 0 ]; then \
 		echo "error\t$(NITRITE_CMD) should have failed without -allowdebug"; \
 		exit 1; \
 	fi
 	@echo "ok\t$(NITRITE_CMD)"
 
 
-some_recipe:
-	@echo $(shell echo 'doing stuff'; exit 123)
-	@echo 'command exited with $(.SHELLSTATUS)'
-	@exit $(.SHELLSTATUS)
-
```

Comment:
I got the impression this is an artifact from you learning about `.SHELLSTATUS` and is no longer needed